ui/src/app/pages/register/register.component.ts

```diff
@@ -88,12 +89,17 @@ export class RegisterComponent extends NbRegisterComponent implements OnInit {
         company,
       }).subscribe(
         respReg => {
+          const first_name = this.user.fullname.split(' ')[0];
+          const last_name = this.user.fullName.replace(`${first_name} `, '');
+          _ps.getByKey(this._groupKey).send('updated', { custom_data: { first_name, last_name } });
```

Comment:
@iatdaitan Instead of disabling auto sending contract acceptance, I update user info upon register.

Optionally, I can disable all events and manually call the endpoint for acceptance at this point.

Reply:
Nice change.